src/components/Card/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+interface CardProps {
+  title: string,
+  text: string,
+  children?: React.ReactNode,
+  // data-test?: string
```

Comment:
It would be good to remove comments before committing files.

Reply:
Removing it :)